src/OpenRiaServices.Server.EntityFrameworkCore/Framework/EFCoreTypeDescriptionContext.cs

```diff
@@ -44,7 +44,7 @@ public IModel Model
             }
         }
 
-        public IEntityType GetEntityType(Type type) => Model.FindEntityType(type);
+        public IEntityType GetEntityType(Type type) => type?.FullName != null ? Model.FindEntityType(type) : null;
```

Comment:
Looked into creating an HashSet based on Model.GetEntityTypes(), problem is that Model.GetEntityTypes() returns `IEnumerable<IEntityType>` but we receive an `Type` in GetEntityType.
We could possibly use an `type is IEntityType entityType && _entityTypes.Contains(entityType)` before out

Reply:
If you want to add test, you might considering adding a new test project just for efcore (just following existing naming and place it under /Test/ instead of /Framework/ folder.

I am unsure how easy it is to setup a type with missing Fullname so i think it might be acceptable in this specific scenario to skip adding a test, but you might want to add a comment in the code what specific case it is intended to solve (null FulleName which cause exception in efcore). 
If there are other errors in the future a  try/catch might be needed.